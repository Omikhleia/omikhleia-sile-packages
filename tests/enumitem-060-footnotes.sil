\begin[papersize=a6, class=book]{document}
\script[src=packages/enumitem]
\font[family=Libertinus Serif]% Default Gentium lacks the white circle.
\nofolios
\neverindent
% Test case 060: ensure migrating contents can contain their own lists without interaction.

\begin{itemize}
    \item{Level 1}
    \begin{itemize}
        \item{Level 2\footnote{In footnote...
\begin{itemize}
    \item{Level 1 Ft}
    \begin{itemize}
        \item{Level 2 Ft}
        \begin{itemize}
            \item{Level 3 Ft}
        \end{itemize}
        \item{Level 2 Ft}
    \end{itemize}
    \item{Level 1 Ft}
\end{itemize}            
}}
        \begin{itemize}
            \item{Level 3 First}
        \end{itemize}
        \item{Level 2 Third}
    \end{itemize}
    \item{Level 1 Third}
\end{itemize}

\end{document}
