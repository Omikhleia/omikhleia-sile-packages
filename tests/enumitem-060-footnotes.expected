Set paper size 	297.6377985	419.5275636
Begin page
Mx 	30.4489
My 	55.6452
Set font 	Libertinus Serif;10;400;;normal;;LTR
T	1768 w=3.5100	(•)
Mx 	39.7039
Mx 	5.2800
Mx 	4.4000
Mx 	4.8900
Mx 	4.4700
Mx 	2.6400
T	45 a=5.2800 70 a=4.4700 87 a=4.9700 70 a=4.4700 77 a=2.6400	(Level)
Mx 	64.4073
T	18 w=4.6500	(1)
Mx 	45.4489
My 	67.6452
T	2222 w=3.5100	(◦)
Mx 	54.7039
Mx 	5.2800
Mx 	4.4000
Mx 	4.8900
Mx 	4.4700
Mx 	2.6400
T	45 a=5.2800 70 a=4.4700 87 a=4.9700 70 a=4.4700 77 a=2.6400	(Level)
Mx 	79.4051
T	19 w=4.6500	(2)
Mx 	84.0551
My 	64.6282
Set font 	Libertinus Serif;6.465;400;;normal;;LTR
T	18 w=3.0062	(1)
Mx 	59.4639
My 	79.6452
Set font 	Libertinus Serif;10;400;;normal;;LTR
T	1753 w=5.4800	(–)
Mx 	69.7039
Mx 	5.2800
Mx 	4.4000
Mx 	4.8900
Mx 	4.4700
Mx 	2.6400
T	45 a=5.2800 70 a=4.4700 87 a=4.9700 70 a=4.4700 77 a=2.6400	(Level)
Mx 	94.4009
T	20 w=4.6500	(3)
Mx 	102.0678
T	39 74 83 84 85 w=18.3400	(First)
Mx 	45.4489
My 	91.6452
T	2222 w=3.5100	(◦)
Mx 	54.7039
Mx 	5.2800
Mx 	4.4000
Mx 	4.8900
Mx 	4.4700
Mx 	2.6400
T	45 a=5.2800 70 a=4.4700 87 a=4.9700 70 a=4.4700 77 a=2.6400	(Level)
Mx 	79.4022
T	19 w=4.6500	(2)
Mx 	87.0705
Mx 	5.9700
Mx 	5.3800
Mx 	2.7100
Mx 	3.6400
Mx 	5.0600
T	53 a=5.9700 73 a=5.3800 74 a=2.7100 83 a=3.7200 69 a=5.0600	(Third)
Mx 	30.4489
My 	103.6452
T	1768 w=3.5100	(•)
Mx 	39.7039
Mx 	5.2800
Mx 	4.4000
Mx 	4.8900
Mx 	4.4700
Mx 	2.6400
T	45 a=5.2800 70 a=4.4700 87 a=4.9700 70 a=4.4700 77 a=2.6400	(Level)
Mx 	64.4041
T	18 w=4.6500	(1)
Mx 	72.0742
Mx 	5.9700
Mx 	5.3800
Mx 	2.7100
Mx 	3.6400
Mx 	5.0600
T	53 a=5.9700 73 a=5.3800 74 a=2.7100 83 a=3.7200 69 a=5.0600	(Third)
Mx 	24.7039
My 	295.3585
Set font 	Libertinus Serif;9;400;;normal;;LTR
T	18 w=4.1850	(1)
Mx 	28.8889
T	15 w=1.9800	(.)
Mx 	48.8689
T	42 79 w=7.5510	(In)
Mx 	59.1379
Mx 	2.7900
Mx 	4.5990
Mx 	4.5360
Mx 	2.8440
Mx 	4.8780
Mx 	4.5360
Mx 	2.8440
Mx 	3.9330
T	71 a=2.7900 80 a=4.5360 80 a=4.5360 85 a=2.8440 79 a=4.8780 80 a=4.5360 85 a=2.8440 70 a=4.0230	(footnote)
T	15 w=1.9800	(.)
Mx 	92.0779
T	15 w=1.9800	(.)
Mx 	94.0579
T	15 w=1.9800	(.)
Mx 	29.8744
My 	306.1585
T	1768 w=3.1590	(•)
Mx 	38.2039
Mx 	4.7520
Mx 	3.9600
Mx 	4.4010
Mx 	4.0230
Mx 	2.3760
T	45 a=4.7520 70 a=4.0230 87 a=4.4730 70 a=4.0230 77 a=2.3760	(Level)
Mx 	60.4363
T	18 w=4.1850	(1)
Mx 	67.3418
T	39 85 w=7.2090	(Ft)
Mx 	43.3744
My 	316.9585
T	2222 w=3.1590	(◦)
Mx 	51.7039
Mx 	4.7520
Mx 	3.9600
Mx 	4.4010
Mx 	4.0230
Mx 	2.3760
T	45 a=4.7520 70 a=4.0230 87 a=4.4730 70 a=4.0230 77 a=2.3760	(Level)
Mx 	73.9348
T	19 w=4.1850	(2)
Mx 	80.8387
T	39 85 w=7.2090	(Ft)
Mx 	55.9879
My 	327.7585
T	1753 w=4.9320	(–)
Mx 	65.2039
Mx 	4.7520
Mx 	3.9600
Mx 	4.4010
Mx 	4.0230
Mx 	2.3760
T	45 a=4.7520 70 a=4.0230 87 a=4.4730 70 a=4.0230 77 a=2.3760	(Level)
Mx 	87.4333
T	20 w=4.1850	(3)
Mx 	94.3357
T	39 85 w=7.2090	(Ft)
Mx 	43.3744
My 	338.5585
T	2222 w=3.1590	(◦)
Mx 	51.7039
Mx 	4.7520
Mx 	3.9600
Mx 	4.4010
Mx 	4.0230
Mx 	2.3760
T	45 a=4.7520 70 a=4.0230 87 a=4.4730 70 a=4.0230 77 a=2.3760	(Level)
Mx 	73.9348
T	19 w=4.1850	(2)
Mx 	80.8387
T	39 85 w=7.2090	(Ft)
Mx 	29.8744
My 	349.3585
T	1768 w=3.1590	(•)
Mx 	38.2039
Mx 	4.7520
Mx 	3.9600
Mx 	4.4010
Mx 	4.0230
Mx 	2.3760
T	45 a=4.7520 70 a=4.0230 87 a=4.4730 70 a=4.0230 77 a=2.3760	(Level)
Mx 	60.4363
T	18 w=4.1850	(1)
Mx 	67.3418
T	39 85 w=7.2090	(Ft)
End page
Finish
